esmvaltool/utils/cmorizers/obs/cmor_config/MTE.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Filename (will be extended by timestamp to match downloaded file)
+filename: 'EnsembleGPP_GL.nc'
+
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: MTE
+  version: 'May12'
+  tier: 3
+  modeling_realm: clim
+  project_id: CMIP5
```

Comment:
Shouldn't project be `OBS`, because this is observational data, not CMIP5? This goes for all cmorizers I guess.

Reply:
In this function `project_id` is used to get the correct CMOR tables, so I guess CMIP5 is correct here.

https://github.com/ESMValGroup/ESMValTool/blob/26816d6274b50815b62782abea8eaca70a9e21f5/esmvaltool/utils/cmorizers/obs/utilities.py#L27-L37

Reply:
Usually `project_id` also ends up as global attribute in the output nc file, can you make sure that it has the correct value there (i.e., `project_id: OBS`)?

Reply:
It does not appear:

```
// global attributes:
                :comment = "" ;
                :history = "Created on 2019-05-08 11:02:15" ;
                :host = "mistralpp4" ;
                :reference = "Jung et al., J. Geophys. Res., 116, doi:10.1029/2010JG001566, 2011." ;
                :source = "http://www.bgc-jena.mpg.de/geodb/BGI/Home" ;
                :tier = "3" ;
                :title = "MTE data reformatted for ESMValTool v2.0a1" ;
                :user = "b309141" ;
                :version = "May12" ;
                :Conventions = "CF-1.5" ;
```

Reply:
> In this function project_id is used to get the correct CMOR tables, so I guess CMIP5 is correct here.

I don't think it is. Custom variables will not work with CMIP5, but do work for OBS.

Reply:
> It does not appear:

It looks like the function that sets the global attributes is not implemented correctly, instead of using all attributes passed in, it just selects a few.